### PR TITLE
[Handlers] Added XML Comments to .NET MAUI Core

### PR DESCRIPTION
**Pull request by jsuarezruiz**
### Description of Change ###

Added XML Comments to .NET MAUI Core (interfaces). 

### Platforms Affected ### 

- Core/XAML (all platforms)

### PR Checklist ###

- [x] Targets the correct branch
- [ ] Tests are passing (or failures are unrelated)

### CODE REVIEW

File: src/Core/src/Core/IPage.cs [hunk 1]
```diff
@@ -1,9 +1,13 @@
...
 {
+	/// <summary>
+	/// A View that occupies the entire screen.
+	/// </summary>
 	public interface IPage
 	{
+		/// <summary>
+		/// Gets or sets the view that contains the content of the Page.
+		/// </summary>
 		public IView View { get; set; }
```
Comment (campersau) [inline src/Core/src/Core/IPage.cs:11]:
```suggestion
		IView View { get; set; }
```

File: src/Core/src/Core/IWindow.cs [hunk 2]
```diff
@@ -1,8 +1,18 @@
...
 {
+	/// <summary>
+	/// Provides the ability to create, configure, show, and manage Windows.
+	/// </summary>
 	public interface IWindow
 	{
+		/// <summary>
+		/// Gets or sets the .NET MAUI Context.
+		/// </summary>
 		public IMauiContext MauiContext { get; set; }
```
Comment (campersau) [inline src/Core/src/Core/IWindow.cs:11]:
```suggestion
		IMauiContext MauiContext { get; set; }
```

File: src/Core/src/Core/IWindow.cs [hunk 3]
```diff
@@ -1,8 +1,18 @@
...
+
+		/// <summary>
+		/// Gets or sets the current Page displayed in the Window.
+		/// </summary>
 		public IPage Page { get; set; }
```
Comment (campersau) [inline src/Core/src/Core/IWindow.cs:16]:
```suggestion
		IPage Page { get; set; }
```

**Review by campersau — COMMENTED** [review]
Unrelated to this PR but interface members are public by default.

**Comment by jsuarezruiz** [comment]
I close this PR, it has been moved to https://github.com/dotnet/maui/pull/419

**Pull request closed without merging**